src/main/java/org/opengis/cite/gpkg12/tiles/TileTests.java

```diff
@@ -302,12 +302,12 @@ public void matrixSetSrsIdReferencesGoodId() throws SQLException
 	 *             If an SQL query causes an error
 	 */
 	@Test(description = "See OGC 12-128r15: Requirement 147")
-	public void featureGeometryColumnsDataValuesSrsId() throws SQLException {
+	public void tileMatrixSetDataValuesSrsId() throws SQLException {
 		try (
 			// 1
 			final Statement statement = this.databaseConnection.createStatement();
 
-			final ResultSet resultSet = statement.executeQuery("SELECT a.srs_id srs_id, a.table_name tn FROM gpkg_tile_matrix_set a, gpkg_contents b WHERE a.table_name = b.table_name and a.srs_id != b.srs_id");
+			final ResultSet resultSet = statement.executeQuery("SELECT a.srs_id srs_id, a.table_name tn FROM gpkg_tile_matrix_set a, gpkg_contents b WHERE a.table_name = b.table_name and a.srs_id != b.srs_id and b.data_type = 'tiles'");
```

Comment:
This isn't going to work because some extensions (e.g., tiled gridded coverages) build on the tiles mechanism but offer a different `data_type`.

Reply:
Yes, I suppose that was kind of the point.  Unless I'm understanding the Tiled Gridded Coverage Data extension incorrectly, it encourages a contents srs id to be a vertical one (i.e. 4979) which may not match the srs in the Tile Matrix Set?
http://docs.opengeospatial.org/is/17-066r1/17-066r1.html#gpkg_contents
If this is the case, then the test fails for gridded coverage data with non matching contents and tile matrix set srs ids.  Or is tile gridded coverage data still required to have the same vertical srs id in the tile matrix set?

Reply:
The tests for coverage data extensions (w/ inherited tile tests) require that:
* The contents srs id always matches the tile matrix set srs id
* The srs table contains an entry for 4979

The tests for coverage data extensions do NOT require that:
* The srs of 4979 is referenced by the contents or tile matrix set

The following coverage data is therefore NOT valid:
* contents srs id of 4979 with an alternate tile matrix set srs id
* contents srs id of any value matching the tile matrix set srs id, but 4979 is not a row in the srs table

The following coverage data is therefore valid:
* contents srs id of 4979 with tile matrix set id 4979
* contents srs id of any value matching the tile matrix set srs id AND 4979 is an unreferenced row in the srs table

Are these the expected testing behaviors indicated by the coverage data extension?